contrib/pyln-client/pyln/client/lightning.py

```diff
@@ -862,7 +862,8 @@ def multifundchannel(self, destinations, feerate=None, minconf=None, utxos=None,
             "destinations": destinations,
             "feerate": feerate,
             "minconf": minconf,
-            "utxos": utxos
+            "utxos": utxos,
+            "best_effort": best_effort
```

Comment:
I changed my mind.  This parameter should be "minchannels", to indicate how many channels are acceptable.  Default is all of them, otherwise it proceeds unless there fewer than "minchannels" channels remaining.  0 is invalid.